include/swift/SIL/SILInstruction.h

```diff
@@ -1806,6 +1806,50 @@ class TailAllocatedDebugVariable {
 static_assert(sizeof(TailAllocatedDebugVariable) == 4,
               "SILNode inline bitfield needs updating");
 
+class SILDebugVariableSupplement {
+protected:
```

Comment:
I realize that this code is all just being refactored — sorry for still asking questions about that code!